### PR TITLE
adding treasurer role to governance document

### PR DESCRIPTION
**Pull request by danielskatz**

### CODE REVIEW

**Comment by JeffCarver** [comment]
Looks good to me.

**Review by cmaimone — APPROVED** [review]

**Review by christophernhill — APPROVED** [review]

**Review by JeffCarver — APPROVED** [review]

File: governance.md [hunk 1]
```diff
@@ -113,6 +113,14 @@ running such elections, and running other votes of the membership as required. T
 will be filled by a Steering Committee member in the first year of their term for a duration of 1
 year.
 
+#### Treasurer
+
+The Treasurer will be responsible for the finances of US-RSE Association. The Treasurer will present an
```
Comment (cosden) [inline governance.md]:
```suggestion
The Treasurer will be responsible for the finances of the US-RSE Association. The Treasurer will present an
```

**Review by cosden — CHANGES_REQUESTED** [review]
LGTM - just the one "the" addition.

**Review by cosden — APPROVED** [review]
Looks good. If we get it fully approved before April 30. Wait to merge until the 30th? Or change the date to the day it's merged?

**Review by lparsons — APPROVED** [review]

**Review by cferenba — APPROVED** [review]

**Comment by danielskatz** [comment]
I guess we would change the two dates to when we merge it - I wrote them as the next SC meeting under that assumption, but it's easy to change.

**Review by jdamerow — APPROVED** [review]

**Review by sandragesing — APPROVED** [review]

File: governance.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 ## Membership and Governance
 
-Date: June 29, 2020
+Date: April 30, 2021
```
Comment (danielskatz) [inline governance.md]:
```suggestion
Date: April 16, 2021
```

File: governance.md [hunk 3]
```diff
@@ -160,5 +168,29 @@ a vote of the membership directly.
 
 ## Ratification
 
-This document was ratified by the following members of the First US-RSE Steering Committee on June
-29, 2020.
+This document, adding the treasurer role, was ratified by the following members of the US-RSE Steering Committee
+on April 30, 2021:
```
Comment (danielskatz) [inline governance.md]:
```suggestion
on April 16, 2021:
```

**Pull request merged**